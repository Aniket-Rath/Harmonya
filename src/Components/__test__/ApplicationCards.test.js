import { render } from '@testing-library/react';
import RenderComp from '../ApplicationCards'

test('renders correctly', () => {
  const { asFragment } = render(<RenderComp />);
  expect(asFragment()).toMatchSnapshot();
});