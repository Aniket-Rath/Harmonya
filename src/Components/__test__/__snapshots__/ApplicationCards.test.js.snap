// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex h-96"
  >
    <div
      class="w-1/2 ml-[20%] mt-[4%]"
      data-aos="fade-up"
      data-aos-duration="600"
    >
      <a
        class="block p-6 max-w-sm bg-white rounded-lg border border-gray-200 shadow-md"
        href="."
      >
        <div>
          <img
            alt="oops"
            class=" mr-[1vh]"
            src="Data.svg"
            width="35vh"
          />
        </div>
        <h5
          class="mb-2 text-3xl font-bold tracking-tight text-gray-900 dark:text-neutral-900"
        >
          Data
        </h5>
        <p
          class="font-normal text-gray-700 dark:text-gray-400 text-lg "
        >
          Automatically augmenting product data with industry and consumer generated concepts.
        </p>
      </a>
    </div>
    <div
      class="w-1/2 mt-[4%]"
      data-aos="fade-up"
      data-aos-duration="600"
    >
      <a
        class="block p-6 max-w-sm bg-white rounded-lg border border-gray-200 shadow-md"
        href="."
      >
        <div>
          <img
            alt="oops"
            class=" mr-[1vh]"
            src="Applications.svg"
            width="35vh"
          />
        </div>
        <h5
          class="mb-2 text-3xl font-bold tracking-tight text-gray-900 dark:text-neutral-900"
        >
          Applications
        </h5>
        <p
          class="font-normal text-gray-700 dark:text-gray-400 text-lg"
        >
          A flexible application suit addressing industry specific needs and use cases.
        </p>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
